### ISSUE TITLE
It give out a .ckpt file to use it locally with our own GPU?

### ISSUE THREAD

**Issue by ZeroCool22** [TheLastBen/fast-stable-diffusion#45]
Or how we can use the training model locally?

thx.

**Comment by TheLastBen** [comment]
you will need the diffusers script to get the model working locally : https://huggingface.co/blog/stable_diffusion

**Comment by 1blackbar** [comment]
this also works , also he managed to bring it down to 10GB GPU
https://github.com/ShivamShrirao/diffusers/tree/main/examples/dreambooth

**Comment by ZeroCool22** [comment]
> this also works , also he managed to bring it down to 10GB GPU https://github.com/ShivamShrirao/diffusers/tree/main/examples/dreambooth

You already tried it on Windows?

**Comment by 1blackbar** [comment]
No xformers wont run on windows

@TheLastBen  where are stored images from gradio ui when i prompt with trained weights? are they stored anywhjere on colab drive ? Also if not , can you save them and use prompt name and seed in the name or something like that ?

**Comment by ZeroCool22** [comment]
> No xformers wont run on windows, also i wouldnt run in localy i do multiple colabs at once my man
> 
> @TheLastBen where are stored images from gradio ui when i prompt with trained weights? are they stored anywhjere on colab drive ? Also if not , can you save them and use prompt name and seed in the name or something like that ?

According to this repo we can train on Windows with WLS + Docker...

**Comment by 1blackbar** [comment]
Oh give it a shot but i dont want to block my own gpu

**Comment by ZeroCool22** [comment]
> Oh give it a shot but i dont want to block my own gpu

Explain me that, how will it block?

**Comment by 1blackbar** [comment]
i cant do anything on my GPU with same speed if its training , come on :)

**Comment by TheLastBen** [comment]
You can run it under windows using docker, preferably  under nvidia docker image

**Comment by ZeroCool22** [comment]
> You can run it under windows using docker, preferably under nvidia docker image

So, i must install this and activate the Virtualization in my BIOS correct?

https://www.youtube.com/watch?v=YozfiLI1ogY

**Comment by TheLastBen** [comment]
I don't think so, just install docker and pull this image : nvcr.io/nvidia/pytorch:22.08-py3
